tests/testCovariance.py

```diff
@@ -0,0 +1,425 @@
+#!/usr/bin/env python
+#
+# LSST Data Management System
+# Copyright 2008-2015 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+#
+
+"""
+Test the basic mechanics of coaddition, coadd processing, and forced photometry.
+
+In this test, we build a mock calexps using perfectly knowns WCSs, with the only sources
+being stars created from a perfectly known PSF, then coadd them, process the coadd (using
+the new measurement framework in meas_base), and then run forced photometry (again, using
+the new forced measurement tasks in meas_base).
+
+We do not check that the results of this processing is exactly what we'd expect, except in
+some cases where it's easy and/or particularly important to do so (e.g. CoaddPsf); we mostly
+just check that everything runs, and that the results make enough sense to let us proceed
+to the next step.
+
+NOTE: if this test fails with what looks like a failure to load a FITS file, try changing
+the REUSE_DATAREPO variable below to False, as sometimes this error message indicates a
+different problem that's revealed when we're not trying to cache the mock data between
+tests (but set REUSE_DATAREPO back to True when done debugging, or this test will be very
+slow).
+"""
```

Comment:
While I know this is just a temporary copy, I think it'd be worthwhile to delete all the tests that aren't actually testing anything new here (and shrink this description accordingly).